Sources/LiveKit/Participant/LocalParticipant.swift

```diff
@@ -378,16 +372,22 @@ public extension LocalParticipant {
                     return try await self._publish(track: localTrack, options: publishOptions)
                 } else if source == .screenShareVideo {
                     #if os(iOS)
+                    
                     let localTrack: LocalVideoTrack
-                    let options = (captureOptions as? ScreenShareCaptureOptions) ?? room._state.roomOptions.defaultScreenShareCaptureOptions
-                    if options.useBroadcastExtension {
+                    let defaultOptions = room._state.roomOptions.defaultScreenShareCaptureOptions
+                    
+                    if defaultOptions.useBroadcastExtension {
+                        if captureOptions != nil {
+                            logger.warning("Capture options for broadcast screen share must be set as room defaults.")
```

Comment:
be more specific with method names etc here and say what will happen instead (which will be e.g. "using room defaults instead")